### PR TITLE
Add supervisor tutorial

### PR DESCRIPTION
**Pull request by ad-daniel**
**Description**
Add basic tutorial for the usage of a supervisor

preview: https://www.cyberbotics.com/doc/guide/tutorial-8-the-supervisor?version=feature-supervisor-tutorial

PS: I don't have Matlab, can somebody test it?
You need to setup up the world as explained in the first section and all the code is available at the end.

### CODE REVIEW

File: docs/guide/tutorial-8-the-supervisor.md [hunk 1]
```diff
@@ -0,0 +1,1143 @@
...
+
+> **Hands-on #4**: Although there are a number of balls available in the Webots library, for this exercise we will be making our own.
+This will allows us to explore the previously mentioned trick.
+1. Reload the simulation.
+2. Add a [Solid](../reference/solid.md) node and give it the `DEF` name `BALL`.
+3. Set the `translation` field of the [Solid](../reference/solid.md) to `1 1 0`.
+4. Double-click the `children` field and add a [Shape](../reference/shape.md) node.
+5. In the `geometry` field add a [Sphere](../reference/sphere.md) node, setting its radius to `0.2`.
+6. Select the `geometry` field and give this node a `DEF` name `SPHERE_GEOMETRY`.
+7. In the `appearance` field add a [PBRAppearance](../reference/solid.md) node and set its `baseColor` field to red (`0 1 0`)
```
Comment (BenjaminDeleze) [inline docs/guide/tutorial-8-the-supervisor.md:672]:
```suggestion
7. In the `appearance` field add a [PBRAppearance](../reference/solid.md) node and set its `baseColor` field to green (`0 1 0`)
```

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
It seems very good and useful to me! Maybe @omichel could read it to review the English style as I am quite bad at it.

It would maybe be better to do the tutorial directly in ENU/FLU ?

It may be useful to add a warning regarding the physic instabilities that can be caused by using the Supervisor to move objects.

**Review by omichel — CHANGES_REQUESTED** [review]
The English looks very good to me.
I just have have a minor suggestion.

**Comment by ad-daniel** [comment]
The problem with doing it with ENU is that it won't touch released branch until December. I was planning on just doing the changes in the `feature-enu-branch` next week when we synchronize the branches, I've kept it simple enough that it will just take a few changes when merging. Being a new file, there won't be any conflicts. What do you think?

**Comment by omichel** [comment]
> What do you think?

Sounds good to me.

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Review by omichel — APPROVED** [review]
I tested the Matlab controller and it works fine.
Thank you.

**Review by BenjaminDeleze — APPROVED** [review]

**Pull request merged**